### PR TITLE
#164069302 Configure PostgreSQL database and settings

### PR DESCRIPTION
**Pull request by NLSanyu**
**What does this PR do?**

* Set up Postgres database by adding Postgres settings in the settings file

**Description of the tasks to be completed?**

* Update the settings file

**How should this be manually tested?**

* Install project requirements using `pip install -r requirements.txt`
* Run the server 

**What are the relevant pivot tracker stores?**
[Delivers #164069302]

### CODE REVIEW

File: authors/settings.py [hunk 1]
```diff
@@ -81,8 +62,12 @@
```
Comment (Stanley-Okwii) [inline authors/settings.py:74]: I like the proper code structure 👍🏿

File: authors/settings.py [hunk 2]
```diff
@@ -1,33 +1,14 @@
...
+from decouple import config, Csv
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
-
-# SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = '7pgozr2jn7zs_o%i8id6=rddie!*0f0qy3$oy$(8231i^4*@u3'
```
Comment (PatrickCmd) [inline authors/settings.py:19]: @NLSanyu I think also this secret_key value should also be exposed to the repository, can you also decouple it.

Comment (NLSanyu) [reply authors/settings.py]: @PatrickCmd May I have more clarification on what you mean by this?

Comment (NLSanyu) [reply authors/settings.py:19]: @PatrickCmd I have worked on it

**Pull request merged**